### DISCUSSION TITLE
What is correct way to run app in production?

### DISCUSSION THREAD

**Discussion by TheWotan** [sidebase/sidebase#94] [Q&A] [answered]
I have an App made with sidebase template. I upload it on DigitalOcean App platform.
And to run it I need to:
 add manually `run command` -> `npm start`
 add manuall `"start": "NODE_ENV=production node .output/server/index.mjs"` to package.json

<img width="1182" alt="Screenshot 2023-01-21 at 16 59 28" src="https://user-images.githubusercontent.com/595124/213872778-5d751035-61ba-42a0-b0f5-1bd9135cede6.png">

So my question is "Are I do everything correctly or do I miss something in the template?"

**Comment by BracketJohn** [comment, marked as answer]
Hey @TheWotan 👋 

Thanks for the question and thanks for using sidebase 🎊 🎊 

Yes, you are doing everything correctly (: Alternatively, you could also do it in one step by adding `NODE_ENV=production node .output/server/index.mjs` into the `run command` field 🪨 

 I think this is something we could add to the Merino template (it's already there in the cheviot one). So I opened an issue for this: https://github.com/sidebase/sidebase/issues/98 It's an easy one, so you could even tackle it yourself if you wanted to! (: 

Please mark this as resolved, if your question was answered 🎊

**Reply by TheWotan** [reply]
thanks, @BracketJohn

**Reply by BracketJohn** [reply]
sure! I also just added the start comman to Merino, so future people will profit from your comment here (:
